trestle/utils/fs.py

```diff
@@ -31,6 +32,10 @@
 
 import yaml
 
+if os.name == 'nt':  # pragma: no cover
+    import win32api
```

Comment:
As discussed these will need to be installed for windows - and I think you use pypiwin32 to get both win32api and win32con

Reply:
I've changed the dependency spec such that this should be okay.